index.html

```diff
@@ -4,8 +4,8 @@
   <head>
     <title>Lambda School Newsfeed</title>
     <link rel="stylesheet" type="text/css" media="screen" href="./CSS/index.css" />
```

Comment:
your file is called `css` but this script tag is pointing to `CSS` so I can't see the styles in browser